Oqtane.Client/Services/ServiceBase.cs

```diff
@@ -173,13 +173,15 @@ public string CreateApiUrl(Alias alias, string serviceName)
         // add entityid parameter to url for custom authorization policy
         public string CreateAuthorizationPolicyUrl(string url, int entityId)
         {
+            string qs = "entityid=" + entityId.ToString();
```

Comment:
What about

```csharp
return url.Contains("?") ? "&" : "?" + "entityid=" + entityId.ToString(); 
```

Reply:
I go for the simple approach.  I know what the code you pasted does, but IMO it's so complex for a one-liner.

Reply:
Not that complex ;) coz the first letter depends on whether the url contains query string or not

Reply:
Both ways are fine for me.  The boss indicated a preference for the longer form.
https://github.com/oqtane/oqtane.framework/pull/155#issuecomment-545851383

Reply:
I prefer inline condition in such case but if the condition is complex or has nested conditions I may prefer if block for readability

Reply:
Thanks @jimspillane I thought I had read that somewhere. Plus at this point I don't want to change a lot of the code, meaning if it's an 'if block', then I will leave it as an if block.

Reply:
This take too long for a tiny change ;) could you please give a meaningful name instead of `qs` and I'm fine with that